### DISCUSSION TITLE
How to install fzf with znap?

### DISCUSSION THREAD

**Discussion by alexandervantrijffel** [marlonrichert/zsh-snap#229] [Q&A]
I am in the process of replacing zplug with zsh-snap. I migrated about half of the plugins and I already got a much lower start up time of my terminal sessions.

## Describe the bug
I'm trying to convert the following zplug configuration to install and load fzf with zsh-snap:

```zsh
zplug "junegunn/fzf", \
  as:command, \
  hook-build: "$ZPLUG_REPOS/junegunn/fzf/install --no-fish --no-bash --bin", \
  use:"bin/*", \
  hook-load: """
source $ZPLUG_REPOS/junegunn/fzf/shell/completion.zsh
source $ZPLUG_REPOS/junegunn/fzf/shell/key-bindings.zsh

  """, \
  defer:3
```

## Steps To Reproduce
I have tried the following commands:

```zsh
znap source junegunn/fzf shell/{completion,key-bindings}.zsh
znap source junegunn/fzf bin/*
znap eval junegunn/fzf "./install --no-fish --no-bash --bin"
```

## Expected behavior
Fzf installs without errors, all files in junegunn/fzf/shell/*.zsh are sourced and executables `fzf` and `fzf-tmux` are available.

## Observed behavior
I'm getting this error when I am using a wildcard:

```
/mydotfiles/fzf/init.zsh:10: no matches found: bin/*
```

And when I run the source command without a wildcard:

```zsh
znap source junegunn/fzf shell/{completion,key-bindings}.zsh
znap eval junegunn/fzf "./install --no-fish --no-bash --bin"
```

These errors are logged:

```
/home/me/.cache/zsh-snap/eval/fzf.zsh:2: command not found: Downloading
/home/me/.cache/zsh-snap/eval/fzf.zsh:cd:3: too many arguments
/home/me/.cache/zsh-snap/eval/fzf.zsh:cd:4: too many arguments
```

So I wonder how I can source files with wildcards, how can I add multiple executable files to the path and how to run the fzf install script without errors? Running `./install --no-fish --no-bash --bin` manually does succeed.

## Environment:
 - OS: Ubuntu
 - Terminal: Alacritty
 - Zsh version: 5.8
 - Repo version: main 9b8edd8 Handle missing upstream

**Comment by marlonrichert** [comment]
`znap eval <repo> <command>` is a replacement for `eval "$( <command> )". Since `./install` directly installs the software and _does not output code_ that should be `eval`ed, passing it's output to `znap eval` won't work.

Instead, call it like this:
```zsh
znap source junegunn/fzf shell/{completion,key-bindings}.zsh
~[junegunn/fzf]/install --no-fish --no-bash --bin
```
